package.json

```diff
@@ -1,7 +1,14 @@
 {
   "name": "raster",
-  "version": "4",
```

Comment:
Can you make this match https://github.com/rsms/raster/blob/master/_data/info.yml and also update https://github.com/rsms/raster/blob/master/misc/dist.sh to rewrite the version number in package.json (so I don’t have to change the version number in two places.)

Reply:
Alt make the Jekyll website setup read the version from the package.json (also update the shell scripts in misc in that case as they read the yaml file currently)